reference/hoon-expressions/rune/buc.md

```diff
@@ -52,6 +51,84 @@ Irregular: `*` makes `%noun`, `^` makes `%cell`, `?` makes
 
 ## Runes
 
+
+### $| "bucbar"
+
+`[%bsbr p=spec q=hoon]`: structure that satisfies a validator.
+
+##### Expands to
+
+Still need help with this. From hoon.hoon:
+```hoon
+      ::  verify, $|
+      ::
+          {$bsbr *}
+        ^-  hoon
+        ::  push the raw product
+        ::
+        :+  %tsls  relative(mod p.mod)
+        ^-  hoon
+        ::  assert
+        ::
+        :+  %wtbn
+          ::  run the verifier
+          ::
+          [%cnhp [%tsbn $/3 q.mod] $/2]
+        ::  produce verified product
+        ::
+        $/2
+```
+
+
+##### Syntax
+
+Regular: **2-fixed**
+
+##### Discussion
+
+`$|` takes two arguments, a mold and a Hoon expression that is typically a gate
+that is used to validate the structure produced by the mold to ensure that the noun
```

Comment:
As per the above, I think it _needs_ to be a gate?

Reply:
Yes, it needs to be a gate producing a flag/loobean.